packages/backend/src/container-utils.ts

```diff
@@ -44,6 +44,17 @@ export async function getContainerEngine(): Promise<extensionApi.ContainerProvid
   return runningPodmanConnections[0].connection;
 }
 
+// Inspect the image / get more information
+export async function inspectImage(engineId: string, image: string): Promise<extensionApi.ImageInspectInfo> {
+  console.log('Inspecting image: ', image);
```

Comment:
to be removed

Reply:
added to match the other parts of container utils